consensus/polybft/state_test.go

```diff
@@ -0,0 +1,614 @@
+package polybft
+
+import (
+	"bytes"
+	"fmt"
+	"math"
+	"os"
+	"path"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/0xPolygon/pbft-consensus"
+	bls "github.com/0xPolygon/polygon-edge/consensus/polybft/signer"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/hashicorp/go-memdb"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/umbracle/ethgo"
+)
+
+func newTestState(t *testing.T) *State {
+	t.Helper()
+
+	dir := fmt.Sprintf("/tmp/consensus-temp_%v", time.Now().Format(time.RFC3339))
+
```

Comment:
Can we remove these extra lines over here?

Reply:
I think linter was unhappy about this for some reason, and that is why I had to separate them with new lines. I'll try to remove them, and see if it will report something.

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/785/commits/e6ea05a82b32ac463830791a0ab9cf661f5fa87c